src/main/java/mateusz/michal/chat/Service/RegistrationService.java

```diff
@@ -83,12 +94,82 @@ private boolean isEmailIncorrect(String email){
         return !pattern.matcher(email).matches();
     }
 
-    public boolean isPasswordIncorrect(String password) {
+    private boolean isPasswordIncorrect(String password) {
         Pattern pattern = Pattern.compile("(?=.*\\d)(?=.*[a-z])(?=.*[A-Z])" +
                 "(?=.*[\\!\\@\\#\\$\\%\\^\\&\\*])(?!=.*\\s).{8,15}");
         return !pattern.matcher(password).matches();
     }
 
+    private List<MyError> validateUserName(String name){
+        List<MyError> errors = new ArrayList<>();
+        if(isNameNull(name)){
+            errors.add(new MyError(400,RegisterFormErrorCode.NAME_NULL,
+                    "parameter name is null"));
+        } else {
+            if(isNameNotPresent(name)){
```

Comment:
Bardziej pasowałoby IMO `isNameEmpty` albo może nawet `isEmptyString`? - nie używamy wtedy zaprzeczenia w nazwie (może być mylące) no i "not present" (aka. "nieobecne") może oznaczać że wcale nie podano pola `name`